pkg/client/generator.go

```diff
@@ -0,0 +1,216 @@
+package client
+
+import (
+	"fmt"
+
+	kubeinformers "k8s.io/client-go/informers"
+	kubeset "k8s.io/client-go/kubernetes"
+	appsset "k8s.io/client-go/kubernetes/typed/apps/v1"
+	batchset "k8s.io/client-go/kubernetes/typed/batch/v1beta1"
+	coreset "k8s.io/client-go/kubernetes/typed/core/v1"
+	rbacset "k8s.io/client-go/kubernetes/typed/rbac/v1"
+	"k8s.io/client-go/rest"
+	restclient "k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/klog"
+
+	configset "github.com/openshift/client-go/config/clientset/versioned"
+	configsetv1 "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
+	configinformers "github.com/openshift/client-go/config/informers/externalversions"
+	regopset "github.com/openshift/client-go/imageregistry/clientset/versioned"
+	regopinformers "github.com/openshift/client-go/imageregistry/informers/externalversions"
+	routeset "github.com/openshift/client-go/route/clientset/versioned"
+	routesetv1 "github.com/openshift/client-go/route/clientset/versioned/typed/route/v1"
+	routeinformers "github.com/openshift/client-go/route/informers/externalversions"
+
+	"github.com/openshift/cluster-image-registry-operator/defaults"
+	"github.com/openshift/cluster-image-registry-operator/pkg/parameters"
+)
+
+type Generator struct {
```

Comment:
Please give it another name, it's terrible to have the same meaningless name in two packages.